src/Mod/BIM/ArchWindowTaskPanel.py

```diff
@@ -0,0 +1,746 @@
+            if index.data():
+                editor.setCurrentIndex(ArchWindow.WindowPartTypes.index(index.data()))
+        elif row == 2:
+            idx = editor.findText(index.data(), QtCore.Qt.MatchExactly)
+            if idx >= 0:
+                editor.setCurrentText(index.data())
+        elif row == 3:
+            if index.data():
+                if index.data() in self.base_objects:
+                    idx = self.base_objects.index(index.data())
+                    editor.setCurrentIndex(idx)
+        elif row in [5, 6]:
+            if index.data():
+                editor.setText(index.data().replace("+V", ""))
+                if "+V" in index.data():
+                    editor.setCheckState(QtCore.Qt.Checked)
+                else:
+                    editor.setCheckState(QtCore.Qt.Unchecked)
+            else:
+                editor.setCheckState(QtCore.Qt.Unchecked)
+        elif row == 8:
+            if index.data():
+                editor.setCurrentIndex(ArchWindow.WindowOpeningModes.index(index.data()))
+
+    def setModelData(self, editor, model, index):
+        """Saves the changed data"""
+
+        row = index.row()
+        if row == 0:
+            model.setData(index, editor.text())
+        elif row in [4, 7]:
+            # this is set directly by the appropriate tools
+            model.setData(index, editor.text())
+        elif row == 1:
+            model.setData(index, ArchWindow.WindowPartTypes[editor.currentIndex()])
+        elif row == 2:
+            model.setData(index, editor.currentText())
+        elif row == 3:
+            idx = editor.currentIndex()
+            if idx > 0:
+                model.setData(index, self.base_objects[idx])
+        elif row in [5, 6]:
+            val = editor.text()
+            if editor.checkState():
+                val += "+V"
+            model.setData(index, val)
+        elif row == 8:
+            model.setData(index, ArchWindow.WindowOpeningModes[editor.currentIndex()])
+        self.task.edit_properties()
+
+    def get_component_names_minus(self, windowparts, name):
+        """Returns all the compoonent names minus the given one"""
+
+        cnames = ["",]
+        for cnum in range(int(len(windowparts)/5)):
+            cname = windowparts[cnum*5]
+            if cname != name:
+                cnames.append(cname)
+        return cnames
+
+    def on_click_wires(self):
+        """Select wires, or opens the componenbt definition dialog"""
+
+        if not self.task.obj.Base:
+            self.on_define()
+            return
+        menu = QtGui.QMenu()
+        if self.task.obj.Base:
+            wires = ["Wire"+str(i) for i in list(range(len(self.task.obj.Base.Shape.Wires)))]
+            for w in wires:
+                act = menu.addAction(w)
+                act.triggered.connect(self.on_select_wire)
+                act.hovered.connect(self.on_hover_wire)
+        act_define = menu.addAction(translate("BIM", "Define..."))
```

Comment:
```suggestion
        act_define = menu.addAction(translate("BIM", "Define…"))
```
Using ellipsis character